internal/cli/atlas/quickstart/prompt.go

```diff
@@ -15,12 +15,18 @@
 package quickstart
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/mongodb/mongocli/internal/usage"
 )
 
+const (
+	mongoshHelp         = "MongoDB CLI will use the MongoDB shell version provided to allow you to access your deployments."
+	mongoshNotFoundHelp = "MongoDB CLI will store the path in your default profile, type ‘mongocli config’ to change it."
```

Comment:
[q] could we 
a) store it in the current profile (in case I ran quickstart with a -P) 

or b)make it a global setting? not bound to a profile?

Reply:
Hi @gssbzn, I have sent you a PM on Slack about this but I am going to copy/paste my question here given that you may have missed it:

> I’d like to have some clarification about this comment:
> I don’t understand why I should store these constants in the profile (they are not used there), what would change if run quickstart with -P? we would still ask the user if they want to run mongosh with the path provided in the profile -> I don’t see any difference.
> Thanks!!